forte/processors/data_augment/algorithms/embedding_similarity_replacement_op.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+
+from typing import Tuple
+import numpy as np
+from texar.torch.data import Embedding
+from texar.torch.data import Vocab
+
+from ft.onto.base_ontology import Annotation
+from forte.common.configuration import Config
+from forte.processors.data_augment.algorithms.text_replacement_op \
+    import TextReplacementOp
+from forte.processors.data_augment.utils.utils import l2_norm
+
+__all__ = [
+    "EmbeddingSimilarityReplacementOp",
+]
+
+random.seed(0)
```

Comment:
This may change the random seed when the class is imported, don't add this line here.